conjure-java-core/src/integrationInput/java/com/palantir/product/SafeLongAliasExample.java

```diff
@@ -42,12 +45,12 @@ public int compareTo(SafeLongAliasExample other) {
         return value.compareTo(other.get());
     }
 
-    public static SafeLongAliasExample valueOf(String value) {
+    public static SafeLongAliasExample valueOf(@Safe String value) {
         return of(SafeLong.valueOf(value));
```

Comment:
the bug is that this line used to read `of(@Safe SafeLong.valueOf(value));`, which the formatter didn't like